e2e/encoding_test.go

```diff
@@ -20,6 +21,10 @@ func TestEncoding(t *testing.T) {
 	cc.EmitEvent("setA1", "A", addr0.String(), addr1.String())
 	cc.EmitEvent("setA2", "A", addr1.String(), addr0.String())
 
+	if os.Getenv("CI") != "" {
+		t.Skip("Skipping test")
```

Comment:
This test should be skipped until this is fixed: https://github.com/0xPolygon/polygon-sdk/pull/44/files#r641339927